lib/zendesk_apps_support/validations/requirements.rb

```diff
@@ -52,11 +52,13 @@ def excessive_requirements(requirements)
           ValidationError.new(:excessive_requirements, max: MAX_REQUIREMENTS, count: count) if count > MAX_REQUIREMENTS
         end
 
-        def invalid_user_fields(requirements)
-          user_fields = requirements['user_fields']
-          return unless user_fields
+        def invalid_custom_fields(requirements)
+          user_fields = requirements['user_fields'] || {}
+          organization_fields = requirements['organization_fields'] || {}
+          custom_fields = user_fields.merge(organization_fields)
```

Comment:
not sure about the logic here, is it possible for an org field to clobber a user field? can't we just iterate over the two hashes separately?